File: programação I/aula 4/ex5.c
#include <stdio.h>

int main(void) {
  int n, i;

  do {
    printf("Digite um valor: ");
    scanf("%d", &n);

    for (i = 1; i <= n; i++) {
      printf("%dº - %d\n", i, n);
    }
  } while (n < 500);

  return 0;
}

File: programação II - Python/aula 3/exercícios de fixação/ex5.py
#Faça um Programa que calcule a área de um quadrado, em seguida mostre o dobro desta área para o usuário.

lado = float(input("Digite o valor do lado do quadrado: "))
area = lado*lado

print("O dobro da área do quadrado é equivalente á: {:.2f}m².".format(area*area))
print("Fim da execução do programa.")